libnavigation-core/src/main/java/com/mapbox/navigation/core/telemetry/telemetryevents/TelemetryReroute.kt

```diff
@@ -0,0 +1,32 @@
+package com.mapbox.navigation.core.telemetry.telemetryevents
+
+import android.annotation.SuppressLint
+import android.location.Location
+import android.os.Parcel
+import com.mapbox.android.telemetry.Event
+
+/**
+ * Documentation is here [https://paper.dropbox.com/doc/Navigation-Telemetry-Events-V1--AuUz~~~rEVK7iNB3dQ4_tF97Ag-iid3ZImnt4dsW7Z6zC3Lc]
+ */
+
+// Defaulted values are optional
+
+@SuppressLint("ParcelCreator")
+class TelemetryReroute(
```

Comment:
Same as above `NavigationRerouteEvent`

Reply:
👍

Reply:
NavigationDepartueEvent. In fact all NavigationEvent have been replaced by TelemetryEvent

Reply:
Navigation<Type>Event have been replaced by Telemetry<Type>Event